commit 43851a0a655a3a961f289087bdea989aa32cc028
Author: Luca Beltrame <lbeltrame@kde.org>
Date:   Sun Jul 21 09:21:16 2019 +0200

    Fix build with Qt 5.13
    
    (cherry picked from commit 43e6bbb4721adc83d8b2a77bd51d9efaf7aefd6b)

diff --git a/src/gui/datewidget.cpp b/src/gui/datewidget.cpp
index 724370e2..e6502bb0 100644
--- a/src/gui/datewidget.cpp
+++ b/src/gui/datewidget.cpp
@@ -35,6 +35,7 @@
 #include <QPushButton>
 #include <QHBoxLayout>
 #include <QFrame>
+#include <QDate>
 #include <QEvent>
 #include <QMenu>
 #include <QWidgetAction>
commit d3c56eca11cd059d4348ab6696822c23d85af61a
Author: Robby Stephenson <robby@periapsis.org>
Date:   Sun Jul 21 19:17:35 2019 -0400

    Fix apparent build problem on Jenkins

diff --git a/src/mainwindow.cpp b/src/mainwindow.cpp
index d8348e80..8250c562 100644
--- a/src/mainwindow.cpp
+++ b/src/mainwindow.cpp
@@ -108,6 +108,7 @@
 #include <QMimeType>
 #include <QMenuBar>
 #include <QFileDialog>
+#include <QMetaMethod>
 
 #include <unistd.h>
 
